sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallConnectionAsync.java

```diff
@@ -266,7 +266,6 @@ Mono<Response<TransferCallResult>> transferToParticipantCallWithResponseInternal
 
             TransferToParticipantRequestInternal request = new TransferToParticipantRequestInternal()
                 .setTargetParticipant(CommunicationIdentifierConverter.convert(transferToParticipantCallOptions.getTargetCallInvite().getTarget()))
-                .setTransfereeCallerId(PhoneNumberIdentifierConverter.convert(transferToParticipantCallOptions.getTargetCallInvite().getSourceCallIdNumber()))
```

Comment:
@wangrui-msft , Can you please check if this is intended?

Reply:
yes, confirmed with the SDK team, they removed it in swagger, and .net SDK also has this change merged.